packages/moonstone/Input/Input.module.less

```diff
@@ -132,7 +132,7 @@
 				color: @moon-input-placeholder-color;
 			});
 
-			.focused& {
+			&:focus-within {
```

Comment:
Is this OK to use?  It's not supported in Blink 53. Do we need a fallback or alternative solution for older browsers?  Or, should we document that 3.x can't be used on older Blink?

Reply:
Hmm. Good question. I think we're safe but we should confirm that apps built with 3.0 will not be deployed to older platforms.

Reply:
I'm curious what other JavaScript and CSS features we already use that are not compatible with such an old browser (released in August 2016).

Reply:
We've been pretty careful out of necessity but there is at least one guarded case (e.g. `ResizeObserver`). Not sure how many others though.